### DISCUSSION TITLE
Adding the count of bookmarks in the bookmark tab will be better

### DISCUSSION THREAD

**Discussion by basilmuhammed** [dailydotdev/daily#777] [Feature Requests]
### Adding the bookmark count
I always bookmark articles, sometimes I don't have enough time to read articles so I bookmark them, whenever I got time and then read them one by one. Adding a count will help to know how much is pending or how much I bookmarked.

**Comment by idoshamun** [comment]
Hi @basilmuhammed,

Thanks for the suggestion :)
